docs/installation/instructions.rst

```diff
@@ -229,7 +229,7 @@ pulp_resource_manager::
     WantedBy=multi-user.target
 
 
-pulp_worker@::
+```pulp_worker@```.service::
```

Comment:
why is service outside the backticks here?

Reply:
agh, yup, you're right!